programs/benchmark/Benchmark.cpp

```diff
@@ -617,8 +630,8 @@ int mainEntryClickHouseBenchmark(int argc, char ** argv)
             ("roundrobin",                                                      "Instead of comparing queries for different --host/--port just pick one random --host/--port for every query and send query to it.")
             ("cumulative",                                                      "prints cumulative data instead of data per interval")
             ("secure,s",                                                        "Use TLS connection")
-            ("user",          value<std::string>()->default_value("default"),   "")
-            ("password",      value<std::string>()->default_value(""),          "")
+            ("user,u",        value<std::string>()->default_value(env_user_str.value_or("default")),   "")
+            ("password,p",    value<std::string>()->default_value(env_password_str.value_or("")),          "")
```

Comment:
Client does not have this.
Because of ambiguity of `--port`.
See also: MySQL CLI.